doc/user/content/cloud/account-limits.md

```diff
@@ -9,21 +9,37 @@ menu:
 
 {{< cloud-notice >}}
 
+Materialize Cloud offers two deployment sizes for free trials, with additional processing and RAM available for a premium upgrade. We use AWS as the cloud service provider and the us-east-1 datacenter.
```

Comment:
@msudberg 

More precisely for free trials, newly created deployments are by default size XS (4vCPU; 32GiB) and users will have the ability to 'size up' to S via the UX for free. Moreover, the other aspect of the free trial to note is that users will be capped still at 2 free deployments to start. If users want to 'size up' deployments and/ or create more than 2 deployments, then they will have to contact the Materialize team to tee up the 'orders.'

For [reference](https://aws.amazon.com/ec2/instance-types/r5/), sizing is currently as follows:
XS = 4vCPUs 32GB memory r5.xlarge (default deployment size)
S = 8vCPUs 64GB memory r5.2xlarge (free)
M = 16vCPUs 128GB memory r5.4xlarge (upsell)
L = 32vCPUs 256GB memory r5.8xlarge (upsell)
XL = 64vCPUs 512GB memory r5.16xlarge (upsell)

IMHO '...additional processing and RAM' could be changed to '...additional processing and memory capacity' or something along those lines.

Reply:
Done.